scripts/common/host-packages.sh

```diff
@@ -95,6 +95,22 @@ function dpkg_install_host_packages() {
     _dpkg_install_host_packages "$dir" "$dir_prefix" "${packages[@]}"
 }
 
+function _dpkg_apt_get_status_and_maybe_fix_broken_pkgs() {
+    logStep "Checking package status"
+    if  sudo apt-get check status ; then
+        logSuccess "Status checked successfully. No broken packages were found."
+        return
+    fi
+
+    logWarn "Attempting to correct broken packages by running sudo apt-get install --fix-broken --yes"
+    sudo apt-get install --fix-broken --yes
+    if sudo apt-get check status ; then
```

Comment:
Remove sudo

Reply:
you will need to remove sudo. the script will always be invoked as root or with sudo.